docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+
+services:
+  nginx:
+    image: bitnami/nginx:1.12.1-r1
+    volumes:
+      - ./dev/nginx:/bitnami/nginx/conf/vhosts
+    depends_on:
+      - backend
+      - frontend
+    environment:
+      - VIRTUAL_HOST=fortress.docker
+      - VIRTUAL_PORT=8080
+    ports:
+      - 8080:8080
+  mongo:
+    image: bitnami/mongodb:3.4.9-r1
+  backend:
+    image: angelrb/fortress:latest
+    command: yarn dev
+    #entrypoint: /scripts/entrypoint.sh
+    depends_on:
+      - mongo
+    volumes:
+      - ./:/app
+      - ./scripts:/scripts
+      - /app/node_modules
+    ports:
+      - 5858:5858
+    environment:
+      - NODE_ENV=development
+  frontend:
```

Comment:
do you think we can map the frontend port to the host? It is useful for non-dingy users

Reply:
I think I can map de port but the `webpack-dev-server` config is not exposed so the connection between the frontend and the backend may not work. Anyway, you can access to the frontend through the nginx container.